src/odemis/dataio/tiff.py

```diff
@@ -2722,3 +2752,210 @@ def _convertTFStoOdemisMD(metadata: str) -> dict:
         logging.warning(f"Failed to parse ThermoFisher metadata: {model.MD_DET_TYPE} - {e}")
 
     return md
+
+def _parseZEISSMetadata(metadata) -> dict:
+    """
+    Parses ZEISS metadata to dictionary
+    :param metadata: metadata string
+    :return: (dict) metadata as dictionary
+    """
+    zmd = {}
+    for line in metadata.split("\r\n"):
+
+        # Metadata is in the format:
+        # HEADER
+        # key = value unit
+
+        # Example:
+        # AP_STAGE_AT_X
+        # Stage at X = 48.5090 mm
+
+        # TODO: try to do it similar to tifffile to extract header information too            
+        try:
+            # NOTE: 
+            # Date, Time are separated by : 
+            # everything else is separated by =
+            try:
+                k, v = line.split("=")
+            except:           
+                k, v = line.split(":", maxsplit=1) 
+            k, v = k.strip(), v.strip()
+            zmd[k] = v
+        except Exception as e:
+            continue
```

Comment:
Please, at least log it, with logging.warning(... exc_info=True).
Or is it most probably a header? In that case maybe you could explicitly detect such "one word", and skip it.

Reply:
- added a check for headers (.isupper() as all headers are single word all caps)
- differentiated ValueErrors from other errors. if both "=", and ":" delimeters dont return kv pairs, the line is a single unused metadata value, and can be ignored. 
- Other errors: something else has gone wrong -> log